news/progress.vendor

```diff
@@ -0,0 +1 @@
+Update progress to 1.5
```

Comment:
I would also add a `.feature` news entry with a description of the changes that pip users will notice.

Reply:
Thanks, done in https://github.com/pypa/pip/pull/6319/commits/2ce546e44af7f91610d8a0b9770933aa3d23e2dd